### DISCUSSION TITLE
HW2 Series Error

### DISCUSSION THREAD

**Discussion by gmg17005** [cooperrc/computational-mechanics#94] [General]
Is anyone else getting the following error? My code was running perfectly fine until 4:18pm today, all of a sudden jupyter notebook started glitching out on me completely. Please let me know if the server crashed and if we're gonna loose all of our work again so that we can locally back it up on a computer. 

Thanks

**Comment by gmg17005** [comment]
Forgot to post the error lol:

![image](https://user-images.githubusercontent.com/112644444/191357161-feab5f75-d32b-4592-abc6-472417276448.png)

**Reply by cooperrc** [reply]
If you're getting a `print(...)` `TypeError`, then the function `print` was reassigned to a different variable.
